WcaOnRails/app/helpers/static_pages_helper.rb

```diff
@@ -1,7 +1,7 @@
 module StaticPagesHelper
-  def format_team_members(team_friendly_id)
-    Team.find_by_friendly_id!(team_friendly_id).current_members.includes(:user).order(team_leader: :desc).order("users.name asc").map do |u|
-      u.user.name + (u.team_leader ? " (leader)" : "")
+  def format_team_members(slug)
```

Comment:
It still is as this is actually getting the members for a team, not a committee.